packages/models/package.json

```diff
@@ -13,18 +13,25 @@
     "lint:autofix": "eslint . --ext .ts,.tsx --fix",
     "format:check": "prettier --check src",
     "format:write": "prettier --write src",
-    "start": "node --watch --no-warnings --loader ts-node/esm ./src/index.ts",
-    "build": "tsc"
+    "start": "pnpm run generate-protobuf && node --watch --no-warnings --loader ts-node/esm ./src/index.ts",
+    "build": "pnpm run generate-protobuf && tsc",
+    "generate-protobuf": "mkdirp ./src/gen && npx protoc --ts_opt=eslint_disable --ts_out ./src/gen --proto_path ./proto ./proto/v1/*.proto && tsc-esm-fix --src='src/gen/v1' --ext='.js'"
```

Comment:
come mai è necessario usare `tsc-esm-fix`? i file generati contengono degli import senza estensione .js?

Reply:
si, al momento non ho trovato una soluzione più elegante se non aspettare che sistemino la cosa in protoc. Mi pare ci sia una issue a riguardo